private/PROPOSAL/math/Vector3D.cxx

```diff
@@ -106,6 +106,11 @@ void Vector3D::swap(Vector3D& vector_3d)
     // swap(cylindric_height_,  vector_3d.cylindric_height_);
 }
 
+Vector3D Vector3D::operator+=(const Vector3D& rhs)
```

Comment:
This does not mutate `this`, does it? Not what I would expect from `+=`.

Herb Sutte makes the strong point, that `operator+` should always be implemented via `operator+=`, like this:

```
T& T::operator+=( const T& other ) {
    //...
    return *this;
}

T operator+( T a, const T& b ) {
    a += b;
    return a;
}
```

https://herbsutter.com/2013/05/20/gotw-4-class-mechanics/
This `operator+=` does not even mutate `*this`, right?